insights/parsers/ovs_vsctl_show.py

```diff
@@ -95,8 +88,7 @@ def parse_content(self, content):
         bridge_dict = {}
         port_dict = {}
         for line in content[1:-1]:
-            line = line.strip()
-            prefix, _, value = line.partition(" ")
+            prefix, value = line.strip().split(None, 1)
```

Comment:
@PaulWay if the uuid is on the second line, which happens when there's a blank line at the top of the input then this line will fail when paring the uuid.  Since the parser assume that the first line is always the uuid and the last line is always the version would it make sense to ensure there are no leading or trailing blank lines first?

Reply:
Yeah, I know.  This is a problem with the original parser.  It assumes that the UUID will be on the first line, the version will be on the last line, and doesn't handle '/bin/ovs: command or file not found'.  I didn't want to change the parser in documenting it, to be honest.  What would you prefer to do, @bfahr - should we leave it, fix it up in another PR or do it in this one?

Reply:
But you already changed the code, that is why this error came up when testing with rules.  I think it might be better if you take out your code changes here and limit to only documentation changes, and make the code changes in a separate PR.